Assets/Scripts/Tetromino.cs

```diff
@@ -3,59 +3,178 @@
 using System.Collections.Generic;
 
 class TetType {
-	public int[] map;
+	public int[][] maps;
 	public int width;
 
-	public TetType(int[] map, int width)
+	public TetType(int[][] maps, int width)
 	{
-		this.map = map;
+		this.maps = maps;
 		this.width = width;
 	}
 }
 
 public class Tetromino {
 
-	private static int[] S_LEFT_MAP = {
-		0, 0, 0,
-		1, 1, 0,
-		0, 1, 1
-	};
-
-	private static int[] S_RIGHT_MAP = {
-		0, 0, 0,
-		0, 1, 1,
-		1, 1, 0
-	};
-
-	private static int[] L_LEFT_MAP = {
-		0, 1, 0,
-		0, 1, 0,
-		1, 1, 0
-	};
-
-	private static int[] L_RIGHT_MAP = {
-		0, 1, 0,
-		0, 1, 0,
-		0, 1, 1
-	};
-
-	private static int[] CROSS_MAP = {
-		0, 1, 0,
-		0, 1, 1,
-		0, 1, 0
-	};
-
-	private static int[] STRAIGHT_MAP = {
-		0, 1, 0, 0,
-		0, 1, 0, 0,
-		0, 1, 0, 0,
-		0, 1, 0, 0
-	};
-
-	private static int[] SQUARE_MAP = {
-		1, 1,
-		1, 1
-	};
+	private static int[][] S_LEFT_MAP = {
```

Comment:
Writing out each possible rotation creates a lot of redundant information. I'd prefer it if just the base rotation was used, and the other rotations created on the fly.

Reply:
Okay, I've just used SRS's initial rotations which I think still works correctly.